raisinbread/RB_files/RB_appointment.sql

```diff
@@ -0,0 +1,6 @@
+SET FOREIGN_KEY_CHECKS=0;
```

Comment:
Why are foreign keys turned off?

Reply:
All the RB .sql start with "SET FOREIGN_KEY_CHECKS=0;"